docs/src/pages/customization/themes/themes-zh.md

```diff
@@ -310,32 +349,40 @@ const theme = createMuiTheme({
 
 ## Accessing the theme in a component
 
-You might need to access the theme variables inside your React components. Let's say you want to display the value of the primary color, you can use the `withTheme()` higher-order component to do so. 下面是一个示例：
+You might need to access the theme variables inside your React components.
+Let's say you want to display the value of the primary color, you can use the `withTheme()` higher-order component to do so. Here is an example:
 
 {{"demo": "pages/customization/themes/WithTheme.js"}}
 
 ## Nesting the theme
 
-The theming solution is very flexible, as you can nest multiple theme providers. This can be really useful when dealing with different area of your application that have distinct appearance from each other.
+The theming solution is very flexible, as [you can nest](/css-in-js/advanced/#theme-nesting) multiple theme providers.
+This can be really useful when dealing with different area of your application that have distinct appearance from each other.
 
-{{"demo": "pages/customization/themes/Nested.js"}}
+{{"demo": "pages/customization/themes/ThemeNesting.js"}}
 
-#### A note on performance
+The inner theme will **override** the outer theme.
+You can extend the outer theme by providing a function:
+
+{{"demo": "pages/customization/themes/ThemeNestingExtend.js"}}
 
-The performance implications of nesting the `MuiThemeProvider` component are linked to JSS's work behind the scenes. The main point to understand is that we cache the injected CSS with the following tuple `(styles, theme)`.
+#### A note on performance
 
+The performance implications of nesting the `MuiThemeProvider` component are linked to JSS's work behind the scenes.
+The main point to understand is that we cache the injected CSS with the following tuple `(styles, theme)`.
 - `theme`: If you provide a new theme at each render, a new CSS object will be computed and injected. Both for UI consistency and performance, it's better to render a limited number of theme objects.
 - `styles`: The larger the styles object is, the more work is needed.
 
 ## API
 
 ### `MuiThemeProvider`
 
-This component takes a `theme` property, and makes the `theme` available down the React tree thanks to React context. It should preferably be used at **the root of your component tree**.
+This component takes a `theme` property, and makes the `theme` available down the React tree thanks to React context.
+It should preferably be used at **the root of your component tree**.
 
 You can see the full properties API in [this dedicated page](/api/mui-theme-provider/).
 
-#### 例子
+#### Examples
```

Comment:
Was this wrong? This is supposed to be the Chinese translation.

Reply:
I'm reseting the translations to the English version. @mbrookes I'm wondering if we shouldn't display English only as the translation start to be outdated.

Reply:
@eps1lon Back to your comment. We have a problem right now with the translation workflow. I think that it's important to have the translation a minimum up-to-date. I think that Google translate > outdated documentation.